README.md

```diff
@@ -23,68 +24,93 @@
+<!-- Table of Contents -->
+# :notebook_with_decorative_cover: Table of Contents
 
+- [About the Project](#star2-about-the-project)
+  * [Functionality](#mechanical_arm-functionality)
+  * [Model collection](#file_cabinet-model-collection)
+- [Getting Started](#running-getting-started)
+- [Basic usage](#computer-basic-usage)
+- [Contributing](#wave-how-to-contribute)
+- [License](#warning-license)
+- [Citation](#page_with_curl-citation)
+- [Contributions](#gem-contributions)
 
-Note that you have to use the respective model name (`str`). For example, if you want to use VGG16 from torchvision, use `vgg16` as the model name and if you want to use VGG16 from TensorFlow/Keras, use the model name `VGG16`. You can further specify the model source by setting the `source` parameter (e.g., `timm`, `torchvision`, `keras`).<br>
 
+<!-- About the Project -->
+## :star2: About the Project
+`thingsvision` is a Python package that let's you easily extract image representations from many state-of-the-art neural networks for computer vision. In a nutshell, you feed `thingsvision` with a bunch of images and tell it which neural network you are interested in. `thingsvision` will then give you the  representation of the indicated neural network for each image so that you will end up with one feature vector per image. You can use these feature vectors for further analyses. We use the word `features` for short when we mean "image representation".
 
-For the correct abbreviations of [torchvision](https://pytorch.org/vision/0.8/models.html) models have a look [here](https://github.com/pytorch/vision/tree/master/torchvision/models). For the correct abbreviations of [CORnet](https://github.com/dicarlolab/CORnet) models look [here](https://github.com/dicarlolab/CORnet/tree/master/cornet). To separate the string `cornet` from its variant (e.g., `s`, `z`) use a hyphen instead of an underscore (e.g., `cornet-s`, `cornet-z`).<br>
+:rotating_light: Note: some function calls mentioned in the [paper](https://www.frontiersin.org/articles/10.3389/fninf.2021.679838/full) have been deprecated. To use this package successfully, exclusively follow this `README` and the [Documentation](https://vicco-group.github.io/thingsvision/). :rotating_light:
 
-PyTorch examples:  `alexnet`, `resnet18`, `resnet50`, `resnet101`, `vit_b_16`, `vit_b_32`, `vgg13`, `vgg13_bn`, `vgg16`, `vgg16_bn`, `vgg19`, `vgg19_bn`, `cornet-s`, `clip-ViT`
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Environment Setup
+<!-- Functionality -->
+### :mechanical_arm: Functionality
+With `thingsvision`, you can:
+- extract features for any imageset from many popular networks.
+- extract features for any imageset from your custom networks.
+- extract features for the [THINGS image database](https://osf.io/jum2f/).
```

Comment:
This point is implicitly true from the previous two statements. I guess you want to emphasize how easy it is to do this for the THINGS images. Maybe consider something like "Conveniently/Easily access features for >26,000 images in the THINGS object image database"

Reply:
I changed the sentence a bit.